### ISSUE TITLE
Transaction Display Issues

### ISSUE THREAD

**Issue by CryptoLexDivi** [Divicoin/Divi#38]
# Expected behavior
I expect each tab to display the correct transactions.

# Actual behavior
I click All Transactions and I see a bunch of transactions I did not initiate (but fine).  I clicked Receive tab and I see a transaction I received (fine). I click the Staked tab and I see the same transaction as was on the Received tab. This can be misleading. I have to click on Received again and then back to Staked. 
# Operating system
Place an 'x' in the box next to your operating system
* [ x] Mac OSX 10.13.1
* [ ] Linux Ubuntu 16.04.4
* [ ] Linux Ubuntu 17.04.4
* [ ] Linux Ubuntu 18.04.4
* [x ] Windows 10

# Steps to reproduce the problem

**Comment by gingfinger** [comment]
I have the same issue on OSX build

**Comment by gingfinger** [comment]
Update: I also have the same wallet running on VPS linux install. When I rand desktop OSX for the first time, I used the divi.conf & wallet.dat from the VPS server. SO technically had 2 wallets running at the same time with the same root files. The osx desktop version showed the above error and the VPS didn't. Once I clicked resync on the desktop wallet, the same errors have now shown up on the VPS wallet which has been running for days prior to setup of the desktop one. 2 different instances of the same wallet showing these errors now, with the vps one previously running with no issues
